### ISSUE TITLE
PSNR and UQI Performance of Pretrained Model

### ISSUE THREAD

**Issue by damlakcaoglu** [junyanz/pytorch-CycleGAN-and-pix2pix#1473]
Hello,

Firstly, I am very appreciated to effort you put on the code. I used your pretrained pix2pix model that is trained with facades dataset and try to reach the PSNR and UQI values in several articles that worked on the same dataset. The model gives nearly 15 dB PSNR and 0.22 UQI values tough these metrics are reported as nearly 24 dB PSNR and 0.7 UQI in the papers. If you want to take a look at, one of the papers that I mentioned is _"The effect of loss function on conditional generative adversarial networks"_. What can be the reason for that? How can I optimize the network, if I am supposed to do? 

Thanks for your time.

**Comment by junyanz** [comment]
We didn't report PNSR and UQI in our original pix2pix paper. I recommend that you contact the authors of "The effect of loss function on conditional generative adversarial networks" for more details.
